R/save.R

```diff
@@ -111,7 +111,10 @@ ggsave <- function(filename, plot = get_last_plot(),
     grDevices::dev.off()
     if (old_dev > 1) grDevices::dev.set(old_dev) # restore old device unless null device
   }))
-  grid.draw(plot)
+  if (!is_bare_list(plot)) {
+    plot <- list(plot)
+  }
+  lapply(plot, grid.draw)
```

Comment:
There should be a `grid.newpage()` in between every draw call

Reply:
Yes I would have thought so as well. This would make sense, but `grid.draw.ggplot()` calls `print.ggplot()`, which has the following clause:
https://github.com/tidyverse/ggplot2/blob/8efc700377c6bfb0583909846f7dc235b0122c34/R/plot.R#L196

which gets activated by default.

Reply:
oh, yeah... forgot it was a ggplot object 🙈